lisa/schema.py

```diff
@@ -870,6 +870,10 @@ class NodeSpace(search_space.RequirementMixin, TypedSchema, ExtendableSchemaMixi
         ),
     )
 
+    # Platform may add image features to
+    # keep track of platform requirements.
+    _image_features: Optional[Dict[str, Any]] = None
```

Comment:
The image features should be included in `ImageSchema`, and extended in `AzureImageSchema`. It's not a part of node space like this.